src/utils/functions/uriToHttp.ts

```diff
@@ -9,7 +9,11 @@
 
 // this function takes an input URI and transforms it to a queryable URL
 // the URI must follow https, http, ipfs, or ipns standard
-export default function uriToHttp(uri: string): string {
+
+function uriToHttp(uri: string): string;
+function uriToHttp(uri: string, retry: 'retry'): string[];
```

Comment:
why is it necessary to overload here (and with a string value?), looks like you just want retry as an optional boolean